css/src/components/popover.scss

```diff
@@ -0,0 +1,40 @@
+$popover-background-color: $body-background !default;
+$popover-border: $border-width solid $border-white-high-contrast !default;
+$popover-shadow: $box-shadow-heavy !default;
+$popover-width: 224px !default;
+
+.popover {
+	display: inline-block;
+	position: relative;
+
+	.popover-summary {
+		color: $hyperlink;
+		list-style: none;
+		cursor: pointer;
+	}
+
+	.popover-content {
+		position: absolute;
+		min-width: $popover-width;
```

Comment:
Think we need to set a reasonable maximum too (or just `width`). See what happens if there's a long string of single line text? (We'll need to do something like this for breadcrumb overflow, for example).

<img width="916" alt="image" src="https://user-images.githubusercontent.com/30843002/145498360-efe6e7b6-d56c-4193-8bc0-04524dbef340.png">

Reply:
I don't know why the rule is `min-width` when it should be just `width`. Probably a mistake, updated! Thanks for the catch 👍 Also, added the `overflow: hidden` here.